index.html

```diff
@@ -517,7 +517,7 @@ <h3>BitstringStatusListCredential</h3>
                 <td>credentialSubject.encodedList</td>
               <td>
 The `encodedList` property of the credential <a>subject</a> MUST be
-the GZIP-compressed [[RFC1952]], base-64 encoded [[RFC4648]] bitstring values
+the GZIP-compressed [[RFC1952]], base64url-encoded [[RFC4648]] bitstring values
```

Comment:
Alternatively here and elsewhere:

```suggestion
the GZIP-compressed [[RFC1952]], base64url-no-pad-encoded [[RFC4648]] bitstring values
```